android/src/main/java/com/swmansion/rnscreens/ScreenViewManager.kt

```diff
@@ -143,6 +140,14 @@ class ScreenViewManager : ViewGroupManager<Screen>(), RNSScreenManagerInterface<
         view.nativeBackButtonDismissalEnabled = nativeBackButtonDismissalEnabled
     }
 
+    private fun logNotAvailable(propName: String) {
+        Log.w("RN SCREENS", "$propName prop is not available on Android")
+    }
```

Comment:
I think this should be moved out from the class to some kind of utils. This time I followed [example of `ScreenStackHeaderConfigViewManager`](https://github.com/software-mansion/react-native-screens/blob/844295039e84697ba25341a43f2fbb50e5c7fb85/android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfigViewManager.kt#L160-L175)